circle.yml

```diff
@@ -1,4 +1,7 @@
 dependencies:
+  pre:
+    # https://github.com/meteor/docs/blob/version-NEXT/long-form/file-change-watcher-efficiency.md
+    - echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```

Comment:
This didn't actually work because `fs.inotify.max_user_watches` is not a key that can be modified (even as root) in a Circle CI container. I would revert this commit, but it doesn't seem to do any harm. The tests still run.